### PR TITLE
DEV-1250 - setAdmins

### PR DESCRIPTION
**Pull request by grp06**
The latest update from this PR:

- we've gone back to using `setAdmin(address[] admins, bool[] isActive)`
- we will use this to activate and deactivate admins, no need to have a separate `add` and `remove` function
- tests updated accordingly

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have updated the documentation accordingly. (if applicable)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/Raft.sol [hunk 1]
```diff
@@ -118,6 +118,10 @@ contract Raft is
     address[] memory admins,
     bool[] memory isActive
   ) public virtual {
+    require(
+      admins.length > 0 && isActive.length > 0,
```
Comment (rahulrumalla) [inline src/Raft.sol:122]: They also have to be of equal length, right?

Comment (rahulrumalla) [reply src/Raft.sol:122]: Could u adjust error message accordingly. It can be something generic as "invalid inputs or lengths" for example

Comment (stefan-girlich) [reply src/Raft.sol:122]: @rahulrumalla @grp06 Why would `addAdmins()` still feature a flag `isActive`? I thought `addAdmins`/`removeAdmins` would remove the need for a separate flag for distinguishing "add"/"remove"?

**Review by stefan-girlich — CHANGES_REQUESTED** [review]

File: src/Raft.sol (hunk 1 above)
Comment (grp06) [reply src/Raft.sol:122]: > 

yes - that's like 4 lines below where you commented. But based on Stefan's comment correctly points out, we won't need that

**Pull request merged**